### PR TITLE
Quadrat: Update Pullquote styles

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
Updates pullquote styles to match https://github.com/Automattic/themes/issues/3743

Before:
Screenshot 2021-05-07 at 11 18 26

After:
Screenshot 2021-05-07 at 11 18 11

### CODE REVIEW

File: blank-canvas-blocks/sass/blocks/_pullquote.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@
 			font-size: var(--wp--custom--pullquote--citation--typography--font-size);
 			font-family: var(--wp--custom--pullquote--citation--typography--font-family);
 			font-style: var(--wp--custom--pullquote--citation--typography--font-style);
+			font-weight: var(--wp--custom--pullquote--citation--typography--font-weight);
```
Comment (MaggieCabrera) [inline blank-canvas-blocks/sass/blocks/_pullquote.scss:20]: Does this break for BCB and the other children that don't have this defined on their theme.json files? Maybe we should default it on BCB's theme.json to whatever the editor/browser has by default

Comment (pbking) [reply blank-canvas-blocks/sass/blocks/_pullquote.scss:20]: It appears to fail gracefully (the missing variable just seems to roll the property back to the browser-defined value of 400.

However it does seem dirty that BCB's CSS would use a variable that it doesn't express.  I suggest that we default that to 400 in BCB.

Additionally this scenario is EXACTLY what I'm excited for by styling elements of blocks in theme.json.  Once that expands beyond anchors and paragraphs we'll be able to remove that ponyfill from BCB entirely.

**Comment by pbking** [comment]
The lack of left/right padding makes the colored-background-variant seem crowded to me:

image

Should it get a css-driven exception?

**Comment by scruffian** [comment]
I pushed a change to BCB to only reset the padding/margin for the blockquote element when you don't choose a solid style.

File: blank-canvas-blocks/sass/blocks/_pullquote.scss [hunk 2]
```diff
@@ -26,3 +25,10 @@
 		color: var(--wp--custom--color--background);
 	}
 }
+
+.wp-block-pullquote:not(.is-style-solid-color) {
```
Comment (pbking) [inline blank-canvas-blocks/sass/blocks/_pullquote.scss]: Is that how we would expect Gutenberg to deal with the scenario?  It feels theme-specific to me and thus no business of BCB's.  Another theme might have a preference for the blockquote paddings to be the same for both and there would be no way to do that.
Or I guess they could with CSS.  But as a polyfill to work like we expect Gutenberg to work what is expected?

Comment (danieldudzic) [reply blank-canvas-blocks/sass/blocks/_pullquote.scss]: @pbking I agree with this being too opinionated for BCB. How about this approach? #3790

Comment (scruffian) [reply blank-canvas-blocks/sass/blocks/_pullquote.scss]: These rules are already in BCB, this change just makes them less powerful. I'm happy to remove them from BCB but we need to check the impact on the other child themes if we do.

Comment (pbking) [reply blank-canvas-blocks/sass/blocks/_pullquote.scss]: I actually meant something more like [this change](https://github.com/Automattic/themes/pull/3780/commits/9e64988e7071053924415f91e2545cc4dfaedd30), where BCB is relatively ignorant of any deviations in style from regular to filled-background.  The [exception is handled](https://github.com/Automattic/themes/pull/3780/files#diff-cb4e625b42338af0b0a25a9effda1fc71e03ada4a095e25c4886f4de88e86726R1) in the child theme with a simple style declaration:

**Comment by pbking** [comment]
Oops!  Looks like you fixed up one of my rebase remessups.  Thanks! 

This looks ready to 🚢  to me.

**Comment by jffng** [comment]
I made some additional refinements to the font size and style:

- Uses a variable font size (same as the "Huge" font-size preset) so that it scales down on mobile
- Line height to 1.4 — 1.3 felt a bit too tight

What do you think @beafialho ?

Desktop | Mobile
------- | ------
Screen Shot 2021-05-11 at 12 49 51 PM | Screen Shot 2021-05-11 at 12 49 37 PM

**Comment by beafialho** [comment]
It looks good @jffng, thanks.

**Pull request merged**